packages/cadl-ranch-specs/http/resiliency/srv-driven-1/mockapi.ts

```diff
@@ -1,137 +1,30 @@
- */
-Scenarios.DPGAddOptionalInput = passOnSuccess(
-  mockapi.get("/servicedriven/parameters", (req) => {
-    if (req.query["parameter"]) {
-      return {
-        status: 200,
-        body: json({ message: `An object was successfully returned` }),
-      };
-    } else {
-      return {
-        status: 400,
-        body: json({ message: `Expected required parameter "parameter"` }),
-      };
-    }
-  }),
+Scenarios.Resiliency_ServiceDriven_V1_AddOptionalParams_fromOneOptional = passOnSuccess(
+  AddOptionalParams_fromOneOptional.v1,
 );
 
-/**
- * Initially has one required query parameter and one optional query parameter.  After evolution, a new optional query parameter is added
- */
-Scenarios.DPGAddOptionalInput_RequiredOptionalParam = passOnSuccess(
-  mockapi.put("/servicedriven/parameters", (req) => {
-    if (req.query["requiredParam"]) {
-      return {
-        status: 200,
-        body: json({ message: `An object was successfully returned` }),
-      };
-    } else {
-      return {
-        status: 400,
-        body: json({ message: `Expected required parameter "requiredParam"` }),
-      };
-    }
-  }),
-);
+Scenarios.Resiliency_ServiceDriven_V1_addContentType = passOnSuccess(AddContentType.v1);
 
-/**
- * Initially has one optional query parameter. After evolution, a new optional query parameter is added
- */
-Scenarios.DPGAddOptionalInput_OptionalParam = passOnSuccess(
-  mockapi.get("/serviceDriven/moreParameters", (req) => {
-    return {
-      status: 200,
-      body: json({ message: `An object was successfully returned` }),
-    };
-  }),
-);
-
-/**
- * A new body type is added (was JSON, and now JSON + JPEG).
- */
-Scenarios.DPGNewBodyTypeJSON = passOnSuccess(
-  mockapi.post("/servicedriven/parameters/json", (req) => {
-    switch (req.headers["content-type"]) {
-      case "application/json":
-        req.expect.bodyEquals({ url: "http://example.org/myimage.jpeg" });
-        return { status: 200 };
-      default:
-        throw new ValidationError("Should be image/jpeg or application/json", {}, req.headers["content-type"]);
-    }
-  }),
-);
-
-/**
- * A new body type is added (was JSON, and now JSON + JPEG).
- */
-Scenarios.DPGNewBodyTypeJPEG = passOnSuccess(
-  mockapi.post("/servicedriven/parameters/jpeg", (req) => {
-    switch (req.headers["content-type"]) {
-      case "image/jpeg":
-        // req.expect.rawBodyEquals("binary");
-        return { status: 200 };
-      default:
-        throw new ValidationError("Should be image/jpeg or application/json", {}, req.headers["content-type"]);
-    }
-  }),
-);
-
-/**
- * Initially the path exists but there is no delete method. After evolution this is a new method in a known path
- */
-Scenarios.DPGAddNewOperation = passOnSuccess(
-  mockapi.delete("/servicedriven/parameters", (req) => {
+Scenarios.Resiliency_ServiceDriven_V1_breakTheGlass = passOnSuccess(
```

Comment:
here i've added a 'break the glass' scenario that we want to test. Basically can a client call an endpoint on the service even if it's not a generated operation in the SDK. I'm using the "new operation" we added in the second deployment of the service for `api-version` `v2`. To me this is fair because that is a scenario for break the glass: "i have an old client but i want to call a new endpoint with a new api version". Thoughts?